tests/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -1677,9 +1677,16 @@ public function testIntrospectReservedKeywordTableViaListTableDetails(): void
         $this->createReservedKeywordTables();
 
         $user = $this->schemaManager->introspectTable('"user"');
-        self::assertCount(2, $user->getColumns());
+        self::assertCount(3, $user->getColumns());
         self::assertCount(2, $user->getIndexes());
         self::assertCount(1, $user->getForeignKeys());
+
+        if ($this->connection->getDatabasePlatform() instanceof SqlitePlatform) {
+            return; // SQLite does not support table / column comment
+        }
```

Comment:
I had to ignore Sqlite as this platform does not support comments.

Reply:
Actually setting and retrieving comments work on sqlite, not sure about the issue:
```php
<?php

declare(strict_types=1);

require __DIR__ . '/vendor/autoload.php';

use Doctrine\DBAL\DriverManager;

$connection = DriverManager::getConnection(['driver' => 'pdo_sqlite']);

$connection->executeStatement(<<<'SQL'
CREATE TABLE "user" -- abc
(
    "user" INT -- def
)
SQL
);

$schemaManager = $connection->createSchemaManager();
$schema = $schemaManager->introspectSchema();
$table = $schema->getTable('user');
var_dump('table comment', $table->getComment()); // => abc
$id = $table->getColumn('user');
var_dump('column comment', $table->getColumn('user')->getComment()); // => abc
```

Reply:
So I reverted this change but it looks like I cannot retrieve the previous run to diagnose the issue. I cannot reproduce the issue locally:
```
XDEBUG_MODE=off vendor/bin/phpunit -c ci/github/phpunit/sqlite3.xml --filter testIntrospectReservedKeywordTableViaListTables
PHPUnit 9.6.22 by Sebastian Bergmann and contributors.

SSSSS.                                                              6 / 6 (100%)

Time: 00:00.206, Memory: 34.00 MB

OK, but incomplete, skipped, or risky tests!
Tests: 6, Assertions: 6, Skipped: 5.
```

Reply:
I fixed this in `\Doctrine\DBAL\Schema\SqliteSchemaManager::getCreateTableSQL` let's discuss the change there.